.terraform-version

```diff
@@ -0,0 +1 @@
+0.10.1
```

Comment:
Hmm, isn't required, just supported, I added this file for convince, but could remove until we actually have a feature that requires it.  I'll update the readme to that affect.